Gemfile

```diff
@@ -33,6 +33,8 @@ else
   end
 
   case ENV['RAILS_VERSION']
+  when /^5.[12]/, /^6.0/
+    gem 'sass-rails', '~> 5.0'
```

Comment:
I'm not sure about this rule.

Reply:
Just following Blacklight's lead here. Setting this gets the Rails 5.2.3 build to pass CI.
https://github.com/projectblacklight/blacklight/blob/master/Gemfile#L38

Reply:
ah! ok thanks!